src/bb-library/Box/Validate.php

```diff
@@ -51,7 +51,7 @@ public function isSldValid($sld)
 
     public function isEmailValid($email, $throw = true)
     {
-        $valid = (preg_match("/[-a-zA-Z0-9_.+]+@[a-zA-Z0-9-]{2,}\.[a-zA-Z]{2,}/", $email) > 0) ? true : false;
+        $valid = (filter_var($email, FILTER_VALIDATE_EMAIL)) ? true : false;
```

Comment:
filter_var does not accept id domains (But the old code either)

So info@idn-tést.eu will fail

Reply:
Well. That's an oversight. I wonder why they haven't updated it to accept those?

Reply:
the accept the puny code variant... 

xn--idn-tst-fya.eu

For example..

Reply:
@jaapmarcus just updated to use `idn_to_ascii`, so that should resolve that issue
https://www.php.net/manual/en/function.idn-to-ascii.php